cmd/bnsd/app/codec.proto

```diff
@@ -131,6 +139,9 @@ message ProposalBatchMsg {
       gov.UpdateElectorateMsg update_electorate_msg = 77;
       gov.UpdateElectionRuleMsg update_election_rule_msg = 78;
       gov.TextResolutionMsg text_resolution_msg = 79;
+      username.RegisterTokenMsg register_token_msg = 80;
+      username.ChangeTokenOwnerMsg change_token_owner_msg = 81;
+      username.ChangeTokenTargetMsg change_token_target_msg = 82;
```

Comment:
I'm still not convinced that when we have potentially several targets for each nft it has to be done via batch, because we might hit a limitation here, especially when there are other transactions in the same batch. 

While it looks like a neat trick to simplify the NFT logic, I'd prefer to have slices in these messages. That allows for more api flexibility.

Reply:
Is what was implemented in https://github.com/iov-one/weave/pull/771/commits/288a34c4bafd8780ebedba1352d29310f477e28b is enough or you had something else in mind?